src/compiler/diagnosticMessages.json

```diff
@@ -5185,6 +5185,14 @@
         "category": "Message",
         "code": 90053
     },
+    "Export '{0}' from module '{1}'": {
+        "category": "Message",
+        "code": 90054
+    },
+    "Export all non-exported member": {

```

Comment:
I think it'd be better off with

```
    "Export all locals imported from module '{0}'" {
```

Reply:
Hi, Daniel, different  variables may come from different modules.

Reply:
1. Why export all non-exported module members? Why not just the ones that are imported?
2. Maybe `"Export all locals from all imported modules."` ?